### PR TITLE
Change CODEOWNERS

### PR DESCRIPTION
**Pull request by justmurphy**
#  #

## ✅ Pre-approval checklist ##

- [ ] This PR has an informative and human-readable title.
- [ ] Changes are limited to a single goal - _eschew scope creep!_
- [ ] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [ ] All relevant type-of-change labels have been added.
- [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Add a tag or create a release.

### CODE REVIEW

**Comment by justmurphy** [comment]
@mcdonnnj @felddy @jsf9k @dav3r we would like the codeowners approved for coverage purposes

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @ChrisSCISA @justmurphy @KwadwoBCISA
+* @justmurphy @Lcerkov @iainDe
```
Comment (mcdonnnj) [inline .github/CODEOWNERS]:
```suggestion
* @iainDe @justmurphy @Lcerkov
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
We prefer alphabetizing these. Otherwise no problems that I can see.

**Comment by justmurphy** [comment]
@mcdonnnj thanks, change made

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔️

**Pull request merged**